blogposts/2021/dev-tool-time-paul-jolly.md

```diff
@@ -0,0 +1,86 @@
+
+[Commentary](https://github.com/tpope/vim-commentary), which enables him to comment and un-comment things in a way that respects the language in use, is another favorite: “It’s just small things like that, when combined, that make your development experience that much more pleasant.”
+
+Paul took us through a Go-based project, showing off fzf and its ability to symbol search across his entire workspace, including the main module and all its dependencies.
+
+Viewers could hear Paul’s keyboard clicking rapidly as he moved from screen to screen and into and out of definitions. Keyboard shortcuts are his jam: “I’m not a mouse person at all.”
+
+As he wrapped up his examples, Paul said that speed was his most important value: “The only real demands I have of my setup is that it’s nice and fast.”
+
+## CUE: A gentle introduction
+
+Paul is a core contributor to the [CUE project](https://github.com/cuelang/cue) and his passion for it enchanted the audience. He spent the bulk of this episode showing us some of what it could do.
+
+In a future post, we’ll share a tutorial that explains more of what CUE can do. For now, consider this a quick tour of some of Paul’s favorite features.
+
+### What is CUE?
+
+CUE, in Paul’s words, “is an open source language for defining, generating and validating all kinds of data,” including configuration, APIs, database schemas, and code.
+
+[Marcel van Lohuizen](https://twitter.com/mpvl_), an [ex-Googler](https://about.sourcegraph.com/blog/ex-googler-guide-dev-tools/), started CUE in 2018. Marcel, Paul said, has a deep background in configuration management, having been part of the [initial Borg team](https://kubernetes.io/blog/2015/04/borg-predecessor-to-kubernetes/) (Borg is a spiritual predecessor to Kubernetes).
+
+CUE solves a number of pain points, such as dealing with white space in YAML or configuration templating. “If you ever find yourself fighting with white space in YAML,” Paul said, “that is an area that CUE can solve.” Essentially, CUE is trying to solve the problem of configurations at scale. Considering how often non-engineers are in charge of maintaining configuration, Paul explained, CUE is also designed to be approachable and accessible.
+
+The end result is the ability to have a “unified view of your configuration across many different systems, as opposed to different configurations and different tooling for different parts of whatever your system may be.”
+
+### What can CUE do?
+
+In one example, Paul showed how, with CUE, you can have data and schema sitting alongside each other. You can also easily translate between different data and schema encodings. CUE can translate between different formats and create a single source of truth that works nicely: “All of your tooling can then hang off that,” Paul said.
```

Comment:
Can the middle two sentences here be combined into one, or is there a difference between translating between data and schema encodings, and translating between formats?